source/blog/2016-11-28-ember-2-10-released.md

```diff
@@ -0,0 +1,295 @@
+
+---
+
+## Ember Data
+
+Ember Data is the offical data persistence library for Ember.js applications.
+
+### Changes in Ember Data 2.10
+
+Ember Data 2.10 represents the work of more than 10 direct contributors with over
+20 commits.
+
+There have been no user facing changes in Ember Data 2.10. Under the
+hood, [@runspired](https://github.com/runspired) has instrumented
+Ember Data with some internal tools to make it easier to track the
+performance of Ember Data. This instrumentation is stripped from the
+production build of Ember Data and should have no impact on existing
+applications that rely on Ember Data.
+
+[@runspired](https://github.com/runspired) also
+[improved the messaging](https://github.com/emberjs/data/pull/4578)
+around assertions and warnings when an adapter responds with no `id`
+for a record.
+
+For more details on the changes in Ember Data 2.10, please review the
+[Ember Data 2.10.0 release page](https://github.com/emberjs/data/releases/tag/v2.10.0).
+
+### Upcoming changes in Ember Data 2.11
+
+Ember Data 2.11 is already shaping up to be a big release with many
+performance and documentation improvements.
+
+#### HasMany Computed Property Keys
+
+One of the performance improvements included in the Ember Data
+2.11.0-beta.1 release fixes an issue where Ember Data was doing too
+much work when records in a `DS.hasMany()` changed. As a result of
+this extra work, Ember Data would incorrectly invalidate computed
+properties that depended on these `hasMany` relationships. After this
+fix you may need to update computed property keys that depended on
+`record.hasManyRelationshipName` to `record.hasManyRelationshipName.[]`.
+
+#### Deprecations in Ember Data 2.11
+
+Ember Data now issues deprecation warnings for
+`store#serialize()`. Instead, it is recommended that you use
+`record.serialize()` in place of
+`store.serialize(record)`.
+
+`store#lookupAdapter()` and `store#lookupSerializer()` have also been
+deprecated starting in Ember 2.11. These methods were never publicly
+documented. In their place we recommend you use `store#adapterFor()`
+and `store#serializerFor()` respectively.
+
+`store#recordIsLoaded()` is deprecated in favor of
+`store#hasRecordForId()`, as the logic in these two methods has been
+identical since
+[March 2015](https://github.com/emberjs/data/pull/2875).
+
+All of the deprecated methods mentioned above will remain in Ember
+Data until v3.0.0. Until then they will log a deprecation warning to
+encourage you to use the recommended replacement API.
+
+#### Move `ds-improved-ajax` out of `beta` channel
+
+The `ds-improved-feature` implemented in
+[PR #3099](https://github.com/emberjs/data/pull/3099) aims to provide
+public hooks on the `rest` adapter to customize how a request for an
+adapter operation should look like: specify the used HTTP method or
+provide custom HTTP headers per request. This feature has been enabled
+in the beta channel in 2.8-beta and has been enabled in the beta
+channel since.
```

Comment:
Nit: I understand the subtlety this is trying to get across (after reading it a few times), but it is a bit clumsy. Does "This feature has been in beta since Ember Data 2.8" preserve enough of that technicality?